testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/realm/RealmTest.java

```diff
@@ -289,6 +290,7 @@ public void createRealmWithPasswordPolicyFromJsonWithInvalidPasswords() {
     @Test
     public void createRealmWithPasswordPolicyFromJsonWithValidPasswords() {
         RealmRepresentation rep = loadJson(getClass().getResourceAsStream("/import/testrealm-keycloak-6146.json"), RealmRepresentation.class);
+        rep.setId(KeycloakModelUtils.generateId());
```

Comment:
Tthis shoudn't be necessary as well, right? Sorry for not spotting it earlier.

Reply:
Yes, it definitely needs changing. Thanks!